addons/qodot/src/util/qodot_editor_button.gd

```diff
@@ -0,0 +1,44 @@
+extends EditorInspectorPlugin
+class_name QodotEditorButton
+
+func _can_handle(object):
+	if object is TrenchBroomGameConfig:
+		return true
+	if object is QodotFGDFile:
+		return true
+	if object is QodotProjectConfig:
+		return true
+	return false
+
+func _parse_property(object, type, name, hint_type, hint_string, usage_flags, wide):
+	return false
```

Comment:
Accidental early return?

Reply:
yeah, I was testing something and I guess it got left in